### DISCUSSION TITLE
EndEvents and user decisions

### DISCUSSION THREAD

**Discussion by bvfalcon** [imixs/imixs-workflow#747] [Ideas]
In [example JSF application](https://github.com/imixs/imixs-jsf-example) we have a [ticket model](https://github.com/imixs/imixs-jsf-example/blob/master/model-ticket.png) with task "Closed" and "End" event. But user have no appropriate button "End" in the form "Closed".
In case of two events (one of them has type "intermediate catch" and other type of "end": 
![grafik](https://user-images.githubusercontent.com/49613576/122741707-82179900-d285-11eb-88ca-8717258d3ca6.png)
) this can make difficulty and unnecessary complexity of BPMN-model.
May be, it will be good to make accessible for user to choose events of type "end" the same as "intermediate catch" events?

**Comment by rsoika** [comment]
Only the 'Intermediate Catch Evnts' can be triggered by the user. The BPMN end event is just for documentation. And as your Task seems not be a final task the End event makes no sense. 
The process instance (workitem) is always in a status defined by a Task element.
